src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,116 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+};
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}: Props) => {
+  const pagesLength = Math.ceil(total / perPage);
+  const pages = Array.from({ length: pagesLength }, (_, index) => index + 1);
+  const items = Array.from({ length: total }, (_, index) => index + 1);
+
+  return (
+    <>
+      <ul className="pagination">
+        <li className={
+          classNames(
+            'page-item',
+            { disabled: currentPage === 1 },
+          )
+        }
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            onClick={() => {
```

Comment:
it is bad practice to pass anonymous arrow functions in jsx.
create handlers and pass reference here and in other places